pages/contact.vue

```diff
@@ -5,35 +5,26 @@
       :description="$t('contact.description')"
     />
 
-    <div class="max-w-7xl mx-auto px-4 sm:px-6 text-center relative z-10 pb-32">
-      <ul class="grid grid-cols-1 md:grid-cols-3 gap-6">
+    <div class="max-w-7xl mx-auto px-4 sm:px-0 text-center relative z-10 pb-32">
```

Comment:
```suggestion
    <div class="max-w-7xl mx-auto px-4 sm:px-6 text-center relative z-10 pb-32">
```
Please bring back the side padding. Otherwise, there is no space for medium size screens.
<img width="1139" alt="Screenshot 2024-03-27 at 10 01 06 AM" src="https://github.com/AstarNetwork/astarwebsite_v2/assets/33358068/2c90443b-a33d-4bd4-b41d-f56b2ebee2d9">

Reply:
resolved with [7ad6314](https://github.com/AstarNetwork/astarwebsite_v2/pull/354/commits/7ad63144e7cea367f549e60a6ef808ab59c16b97)

Reply:
<img width="1340" alt="image" src="https://github.com/AstarNetwork/astarwebsite_v2/assets/32065632/4ce553af-da16-402d-87ce-fdafbe1853b4">